lib/tasks/order_cycles.rake

```diff
@@ -0,0 +1,19 @@
+namespace :ofn do
+  desc 'destroys the specified order cycle'
+  task :remove_order_cycle, [:order_cycle_id] => :environment do |_task, args|
```

Comment:
Isn't this easier in the admin interface?

Reply:
AFAIK we don't remove the coordinator fees applied to the OC when destroying it. That's why.